changelog.rst

```diff
@@ -3,6 +3,13 @@
 Changelog
 =========
 
+hammr 3.8.6 (2018-08-20)
+------------------------
+
+Evolutions:
+
+* Support generation and publication of machine images for OpenShift (it is no longer possible to publish from a Docker image to OpenShift)
```

Comment:
saying that you now support ... then in parenthesis that you no longer support seems confusing. And if there is an exception should not be in parenthesis as this information is quite important
Support generation and publication of machine images for OpenShift. However, it is not possible to publish from a Docker image to OpenShift.

Reply:
In fact, in reading the other changes, I see that you cannot generate an Openshift image from a Windows template, so this should also be mentioned in change log

Reply:
In fact in last FP we were able to publish from a Docker image, which is no longer possible.